LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Solid
+Copyright (c) 2017 - present
```

Comment:
`- present` is not valid in a copyright statement. Also, a copyright statement should include the legal name of the entity which is claiming that copyright.